Consolidate Dockerfiles

djbingham opened the pull request:
https://eaflood.atlassian.net/browse/PSD-519

- Correct image repository names
- Remove `ENTRYPOINT` from parent images so shell-form commands can be used
- Combine Dockerfiles using multi-stage builds, similar to Node parent image repository

Commit:
Correct assumed image names in example

Commit:
Remove default ENTRYPOINT from parent images

Commit:
Consolidate Dockerfiles and correct image names

Rename sdk and runtime to development and production to match Node
parent image naming.

Commit:
Increment version

johnwatson484 commented:
The image names in the README also need updating to reflect these changes

Commit:
Correct image names in Readme

johnwatson484 commented:
Slightly nitpicky as not sure if in scope of this PR, but the year in the licence should be 2020

johnwatson484 reviewed (commented):
Dockerfiles look good, just needs some minor text changes.

Commit:
Correct license year to 2020

djbingham commented:
Thanks @johnwatson484 

I have:

1. updated the image names in the README
2. removed comments from the Dockerfiles on setting the `ASPNETCORE_ENVIRONMENT` variable - it seems self-evident what that's doing
3. updated the license year to 2020

johnwatson484 commented on README.md:
```diff
@@ -4,8 +4,8 @@ This repository contains .Net Core parent Docker image source code for the Futur
-- `ffc-dotnetcore-runtime`
-- `ffc-dotnetcore-sdk`
+- `ffc-dotnetcore`
+- `ffc-dotnetcore-develpment`
```
Slight spelling mistake

djbingham replied:
Thanks. Corrected.

Commit:
Use awsRegion variable in Jenkinsfile

Commit:
Correct typo in README

johnwatson484 reviewed (approved):
LGTM
